src/corelocation.cs

```diff
@@ -474,6 +474,9 @@ partial interface CLLocationManager {
 		[Export ("stopMonitoringLocationPushes")]
 		void StopMonitoringLocationPushes ();
 
+		[Watch (9,0), NoTV, NoMac, NoiOS, NoMacCatalyst]
+		[Export ("requestHistoricalLocationsWithPurposeKey:sampleCount:completionHandler:")]
+		void RequestHistoricalLocation (string purposeKey, nint sampleCount, Action<NSArray<CLLocation>, NSError> handler);
```

Comment:
Looks like this method could be `[Async]`

Reply:
Good Call!

Reply:
@rolfbjarne addressed!